scrapy/core/downloader/__init__.py

```diff
@@ -59,6 +59,8 @@ def __str__(self):
 def _get_concurrency_delay(concurrency, spider, settings):
     delay = settings.getfloat('DOWNLOAD_DELAY')
     if hasattr(spider, 'download_delay'):
+        warnings.warn("download_delay is deprecated"
```

Comment:
```suggestion
        warnings.warn("The Spider.download_delay attribute is deprecated",
```
In addition to @wRAR's comment, I think the current message doesn't make it clear that what's deprecated is the _spider_ attribute.